### ISSUE TITLE
Shorter label for notifications

### ISSUE THREAD

**Issue by bertob** [bleakgrey/tootle#206]
The "Notifications" string is a bit long, making the entire view switcher very wide (since the longest string is what determines its width):

![image](https://user-images.githubusercontent.com/1908896/88049508-8dce7d00-cb55-11ea-9ccd-e0b26aa34980.png)

A shorter string would look a lot nicer here:

![image](https://user-images.githubusercontent.com/1908896/88049465-814a2480-cb55-11ea-83f1-1d87e696f65d.png)

![image](https://user-images.githubusercontent.com/1908896/88049445-75f6f900-cb55-11ea-98f6-402a09b6bc0a.png)

I'm not sure if "Mentions" is perfect, but it was the first thing that came to mind and I think it could work. There might be other/better options though.

**Comment by bleakgrey** [comment]
"Mentions" is a semantically narrow variant that excludes follow requests and poll results. No other client I'm aware of changes the name of this timeline and I'd like to keep the vanilla Mastodon's terminology the way it is as deviating from it just for the sake of design can cause confusion. 

Besides, even if some other variant works in English, there's no beating "Benachrichtigungen" in German or "Уведомления" in Russian. Trust me, we tried :)

The only way I can think of to compact the tab bar somehow is to drop the labels completely (for instance, Tootle 0.2 used icon tabs), but, again, that would be a deviation from the standard look of the libhandy's ViewSwitcher widget. I'm not even sure if ViewSwitcher supports icon-only mode, I didn't test this use case at all.

Honestly, I don't see a problem here. Libhandy enables a "compact" mode that places tab labels below their icons if no more space is available, and it seems to work just fine for "Notifications".

tl;dr: IMHO, "Notifications" is the universal way to go. Long timelines labels are unavoidable, but libhandy handles them gracefully.
